src/Equinor.ProCoSys.IPO.WebApi/appsettings.json

```diff
@@ -77,12 +77,13 @@
   "FusionRequestTimeout": 110,
   "FusionTotalTimeout": 100,
   "Email": {
-    "Enabled": false,
+    "Enabled": true,
     "From": "",
     "Password": "",
     "EnableSsl": true,
     "Server": "",
-    "Port": 25
+    "Port": 25,
+    "FakeEmail": "false"
```

Comment:
set FakeEmail" "true" as default. Then the production environment will be the only environment we need to set "true", less possibility to accidently send real emails from developers pc, dev and test

Reply:
There has never been any fake mode for the Fusion solution, as far as I can see. This only applies to the SMTP based fallback solution. Anyway, I am changing the default setting for this attribute to true.